Sources/Slacket/Services/BackendService.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  BackendService.swift
+//  Slacket
+//
+//  Created by Jakub Tomanik on 20/05/16.
+//
+//
+
+import Foundation
+
+protocol BackendService {
```

Comment:
Ok, "Backend" word was a bit misleading for me, I thought it's some kind of service :)